library/src/main/java/com/android/volley/toolbox/CacheKey.java

```diff
@@ -0,0 +1,14 @@
+package com.android.volley.toolbox;
+
+public class CacheKey {
```

Comment:
agree with @elihart. Make it maybe `ImageCacheKey` and move the class member to the top